extensions/amp-story/1.0/amp-story.js

```diff
@@ -442,6 +442,14 @@ export class AmpStory extends AMP.BaseElement {
     // hover. (See 17654)
     this.element.removeAttribute('title');
 
+    // Remove text nodes which would be shown outside of the amp-story
+    const textNodes = Array.from(this.element.childNodes).filter(
```

Comment:
I'm not sure if we polyfill `Array.from()`, could we use this custom `childNodes` helper instead?
https://github.com/ampproject/amphtml/blob/master/src/dom.js#L363-L378

Reply:
Done I think :)